### ISSUE TITLE
Double Sim card not supported

### ISSUE THREAD

**Issue by dshitara** [finnmglas/Launcher#89]
I installed this launcher on my Android phone, which has two SIM cards. Because of this I have two separate WhatsApp apps installed (one for each SIM card). However, only one of those appears on this launcher's list, making it impossible to allocate a gesture to the second WhatsApp app.

Is there a solution to this?

**Comment by finnmglas** [comment]
_This issue is likely related / similar to #83_

Is the _second WhatsApp_ a separate app or does it use some third app (like DualMessenger)?

These often do not actually create a second app installation but rather a link that opens the app with a different storage location. If you're using one of those it's likely Launcher does not recognize the app (as Launcher doesn't support links or widgets quite yet).

I am not quite sure how this can be implemented, however, I am sure it can be.

**Comment by dshitara** [comment]
Yes, it's really through Dual Messenger (my phone is a Samsung A50).

So it's really not supported, then? Any plans to include that?

- Denise

**Comment by finnmglas** [comment]
I'd love to include it, however don't currently have the resources (time) to actively introduce new features to Launcher (I study physics, got a research job and challenged myself to read 365 Books this year xD).

Thank you for reaching out however, which makes it way more likely for your issue to be resolved at some point (not in the next week though) : )
